TSPL.docc/GuidedTour/GuidedTour.md

```diff
@@ -381,6 +381,21 @@ the conditional must be a Boolean expression ---
 this means that code such as `if score { ... }` is an error,
 not an implicit comparison to zero.
 
+You can write `if` or `switch`
+after the equals sign (`=`) of an assignment
+or after `return`,
+to choose a value based on the condition.
+
+```swift
+let scoreDecoration = if teamScore > 10 {
+    "🎉"
+} else {
+    ""
+}
```

Comment:
We probably ought to decide on what our preferred styling is here. Personally I don't mind it written like this, though some folks may prefer it written as:

```swift
let scoreDecoration =
    if teamScore > 10 {
        "🎉"
    } else {
        ""
    }
```

Any thoughts @airspeedswift?

Reply:
The consensus from the discussions I've had is to format multiline `if` and `switch` expressions as they are in this PR, starting on the same line as the assignment, and without additional indentation. This follows the general rules that we use elsewhere for formatting multiline constructs.